core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/ambari/AmbariLdapService.java

```diff
@@ -58,6 +59,9 @@ public void setupLdap(Stack stack, Cluster cluster, AmbariRepo ambariRepo) {
             ldapConfigs.put("ambari.ldap.connectivity.anonymous_bind", false);
             if (ambariRepositoryVersionService.isVersionNewerOrEqualThanLimited(ambariRepo::getVersion, AMBARI_VERSION_2_7_2_0)) {
                 ldapConfigs.put("ambari.ldap.advanced.collision_behavior", "convert");
+                if (ambariRepositoryVersionService.isVersionNewerOrEqualThanLimited(ambariRepo::getVersion, AMBARI_VERSION_2_7_100_0)) {
```

Comment:
Do we need this additional check? I think if you  put ambari.ldap.advanced.disable_endpoint_identification into Ambari property then older Ambari are just ignoring this property

Reply:
It is throwing an UnknownProperty error otherwise..